test/LiquidCrystal.cpp

```diff
@@ -37,4 +37,18 @@ unittest(loop) {
   assertEqual("????TANK CONTROL", lines.at(1));
 }
 
+unittest(writeLine) {
+  LiquidCrystal_TC* testLcd = LiquidCrystal_TC::instance();
+  testLcd->writeLine("Hello", 0);
+  testLcd->writeLine("World", 1);
+  std::vector<std::string> lines = testLcd->getLines();
+  assertEqual("Hello           ", lines.at(0));
+  assertEqual("World           ", lines.at(1));
+  try {
+    testLcd->writeLine("this will break", 3);
```

Comment:
Does this cause an error? Should the test include an `assertTrue(false)` if they try continues after the line `testLcd->writeLine("this will break", 3);` 

I am not sure it actually will fail for line 3. Would it not just print this on line 1?

Reply:
In theory it should give an error if you try to write beyond the number of lines. So, yes, there should be an `assertTrue(false)` after this line so we don't just silently continue. Even better would be setting a flag to `false` before making the call and then set the flag to `true` inside the catch, and then after the try/catch, have `assertTrue(flag);`

Reply:
This is a relic from when writeLine used to throw an exception. I have updated the testing to reflect the new behavior adopted by modulo.

Reply:
I think your changes addressed my issue.